test/unit/specs/components/staking/__snapshots__/PageValidator.spec.js.snap

```diff
@@ -206,7 +206,7 @@ exports[`PageValidator has the expected html structure 1`] = `
               My Bonded Steak
             </dt>
             <dd>
-              &lt; 0.01
+              0.00
```

Comment:
is this supposed to be 0 ?

Reply:
It was, I changed it now to something show another value.